run.sh

```diff
@@ -9,13 +9,13 @@ run__prod () {
 run__dev () {
 	# dev mode
 	docker build --file Dockerfile.dev -t root-pythia-dev:latest .
-	docker run --rm --interactive --tty --env-file .env.dev --volume ./src:/opt/root-pythia/src root-pythia-dev:latest
+	docker run --rm --interactive --tty --env-file .env.dev --volume `realpath ./src`:/opt/root-pythia/src root-pythia-dev:latest
```

Comment:
I'll look into, this seems weird...

Reply:
I think it is a problem with my distro, and I found a lot of people were forced to use absolute path. If you find a fix I'm interested tho.

Reply:
I'm going to leave it like that since it still works for everyone that way.